apps/playwright/tests/admin/talent-nomination.spec.ts

```diff
@@ -1,9 +1,234 @@
+      .fill("Sidekick");
+    await appPage.page.getByRole("button", { name: /next step/i }).click();
+
+    // Fill out the nomination details step
+    await appPage.page.getByRole("checkbox", { name: /advancement/i }).click();
+    await appPage.page
+      .getByRole("checkbox", { name: /lateral movement/i })
+      .click();
+    await appPage.page
+      .getByRole("checkbox", { name: /development program/i })
+      .click();
+    await appPage.page
+      .getByRole("textbox", { name: /Search reference's work email/i })
+      .fill("does_not_exist@gc.ca");
+    await appPage.page
+      .getByRole("button", { name: /Search work email/i })
+      .click();
+    await appPage.waitForGraphqlResponse("EmployeeSearch");
+    await expect(
+      appPage.page.getByText(/We couldn't find a matching profile/i),
+    ).toBeVisible();
+    await appPage.page
+      .getByRole("textbox", { name: /Reference’s name/i })
+      .fill("Peter Parker");
+    await appPage.page
+      .getByRole("textbox", { name: /Reference's work email/ })
+      .fill("not_spiderman@gc.ca");
+    await appPage.page.getByRole("combobox", { name: /group/i }).click();
+    await appPage.page.getByRole("option", { name: "IT", exact: true }).click();
+    await appPage.page
+      .getByRole("combobox", { name: /level/i })
+      .selectOption({ label: "2" });
+    await appPage.page
+      .getByRole("combobox", { name: /department or agency/i })
+      .selectOption({ label: "Senate" });
+    await appPage.page
+      .getByRole("group", {
+        name: /lateral movement options/i,
+      })
+      .getByRole("checkbox", { name: /other/i })
+      .click();
+    await appPage.page
+      .getByRole("textbox", { name: /other lateral move option/i })
+      .fill("Right");
+    await appPage.page
+      .getByRole("group", {
+        name: /development program options/i,
+      })
+      .getByRole("checkbox", { name: /other/i })
+      .click();
+    await appPage.page
+      .getByRole("textbox", { name: /other development program option/i })
+      .fill("Sidekick training");
+    await appPage.page.getByRole("button", { name: /next step/i }).click();
+
+    // Fill out the rationale step
+    await appPage.page
+      .getByRole("textbox", { name: /nomination rationale/i })
+      .fill("They are the nominators best friend.");
+    const skillCombobox = appPage.page.getByRole("combobox", {
+      name: /top 3 key leadership competencies/i,
+    });
+    await skillCombobox.fill(`${skill1.name.en ?? ""}`);
+    await skillCombobox.press("ArrowDown");
+    await skillCombobox.press("Enter");
+    await skillCombobox.fill(`${skill2.name.en ?? ""}`);
+    await skillCombobox.press("ArrowDown");
+    await skillCombobox.press("Enter");
+    await skillCombobox.fill(`${skill3.name.en ?? ""}`);
+    await skillCombobox.press("ArrowDown");
+    await skillCombobox.press("Enter");
+    await appPage.page.getByRole("button", { name: /next step/i }).click();
```

Comment:
I wish there was a better way to do this :sweat_smile: 

Do you think it is worth it for me to create a ticket for a helper like `fillCombobox`? :thinking:  Or is not that bad?

Reply:
I'm not sure if it's worth it at this point, I can only find one other place where we do this right now.